src/main/java/picard/fingerprint/CrosscheckReadGroupFingerprints.java

```diff
@@ -50,7 +50,7 @@
 @CommandLineProgramProperties(
         usage = "Checks if all read groups within a set of BAM files appear to come from the same individual",
```

Comment:
actually, this seems to be incorrect: "a set of..." program only takes a single file as input.